Tasks/AzureMonitorAlerts/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "azuremonitoralerts",
```

Comment:
I think we should add shrink-wrap. would be better. use same version of npm that the hosted agent has.